packages/xo-server/src/xapi/utils.js

```diff
@@ -381,3 +381,15 @@ export const canSrHaveNewVdiOfSize = (sr, minSize) =>
   sr.content_type !== 'disk' && // removable
   sr.content_type !== 'iso' && // read only
   sr.physical_size - sr.physical_utilisation >= minSize
+
+export async function checkDateConsistency (xapi) {
+  if (
+    Math.abs(
+      parseDateTime(
+        await xapi.call('host.get_servertime', xapi.pool.master)
+      ).getTime() - Date.now()
+    ) > 2e3
```

Comment:
Why 2 seconds?

Do you think it's enough?

Reply:
I don't know. When I tested it, it was always <1s but I don't know if there can be a much bigger difference. Maybe 10s?